Classes/Bookmark v2/BookmarkNavigationController.swift

```diff
@@ -38,28 +38,26 @@ final class BookmarkNavigationController {
             selector = #selector(BookmarkNavigationController.add(sender:))
         }
 
-        let item = UIBarButtonItem(
-            image: UIImage(named: imageName),
-            style: .plain,
-            target: self,
-            action: selector
-        )
+        let itemBtn = UIButton(frame: Styles.Sizes.barButton)
+        itemBtn.setImage(UIImage(named: imageName)?.withRenderingMode(.alwaysTemplate), for: .normal)
+        itemBtn.addTarget(self, action: selector, for: .touchUpInside)
+        let item = UIBarButtonItem(customView: itemBtn)
         item.accessibilityLabel = accessibilityLabel
         return item
     }
 
     // MARK: Private API
 
-    @objc func add(sender: UIBarButtonItem) {
+    @objc func add(sender: UIButton) {
         Haptic.triggerSelection()
-        sender.action = #selector(BookmarkNavigationController.remove(sender:))
-        sender.image = UIImage(named: "nav-bookmark-selected")
+        sender.addTarget(self, action: #selector(BookmarkNavigationController.remove(sender:)), for: .touchUpInside)
+        sender.setImage(UIImage(named: "nav-bookmark-selected")?.withRenderingMode(.alwaysTemplate), for: .normal)
         store.add(model)
     }
 
-    @objc func remove(sender: UIBarButtonItem) {
-        sender.action = #selector(BookmarkNavigationController.add(sender:))
-        sender.image = UIImage(named: "nav-bookmark")
+    @objc func remove(sender: UIButton) {
+        sender.addTarget(self, action: #selector(BookmarkNavigationController.add(sender:)), for: .touchUpInside)
+        sender.setImage(UIImage(named: "nav-bookmark")?.withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
We might need to set button's imagview tint color. Guess it will result in black coloured image. Isn't?

Reply:
The default tint color is the default blue!

Reply:
okay then nvm